app.js

```diff
@@ -301,15 +302,39 @@ function ClientOptionsHandler(fileSources) {
     };
     this.setCompilers = compilers => {
         const blacklistedKeys = ['exe', 'versionFlag', 'versionRe', 'compilerType', 'demangler', 'objdumper',
-            'postProcess'];
+            'postProcess', 'demanglerClassFile', 'isSemVer'];
+        let semverGroups = [];
         const copiedCompilers = JSON.parse(JSON.stringify(compilers));
-        _.each(options.compilers, (compiler, compilersKey) => {
+        _.each(copiedCompilers, (compiler, compilersKey) => {
```

Comment:
Not necessarily worth doing... but, outside of the `_.each()` you could do:

```javascript
const semverGroups = _.chain(copiedCompilers)
   .filter(compiler => compiler.isSemVer)
   .pluck('group')
   .uniq()
   .value();
```

which is kinda cool but a bit underscore-show-offy :grinning:

Reply:
Ouch. This is the kind of cool stuff I wish I could come up without having to burn my brain in the proccess!
If the current approach was not saving a full cycle thru the array again, it would probably be nice to have it